ecl2df/summary.py

```diff
@@ -1,7 +1,11 @@
 """Provide a two-way Pandas DataFrame interface to Eclipse summary data (UNSMRY)"""
 import logging
-import datetime
 from pathlib import Path
+import warnings
+
+# The name 'datetime' is in use by a function argument:
+from datetime import date as datetime_date
+from datetime import datetime as datetime_datetime
```

Comment:
`dt` seems conventional, might be better here?

```python
import datetime as dt
…
```
But definitely unfortunate that `datetime` is a parameter 🤣

Reply:
Changed to `dt`. Planning to remove the datetime parameter in the future.